roster/src/Roster.Core/Domain/AggregateRoot.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace Roster.Core.Domain
 {
-    public abstract class AggregateRoot : IEventSource
+    public abstract class AggregateRoot
```

Comment:
Why removing the `IEventSource` interface implementation?

Reply:
The interface itself was needless, AggregateRoot is an abstract class and provides needed methods. Nobody was actually using `IEventSource`.